.kokoro/presubmit/graalvm-native-a.cfg

```diff
@@ -3,7 +3,7 @@
 # Configure the docker image for kokoro-trampoline.
 env_vars: {
   key: "TRAMPOLINE_IMAGE"
-  value: "gcr.io/cloud-devrel-kokoro-resources/graalvm:22.3.3"
+  value: "gcr.io/cloud-devrel-public-resources/graalvm_a:1.7.1"
```

Comment:
Should these `graalvm-native-x.cfg`s have a dependabot/renovatebot comment to tie the version to native-image-shared-config?

Reply:
Good point, enabled renovate-bot to update the java-shared-config version. Tested at https://github.com/mpeddada1/forking-renovate-repro/pull/8